scripts/fix_doctests.py

```diff
@@ -92,6 +92,11 @@ def adjust(self):
         # Finally, set the flag for location quality
         self.checked_ok = \
             dedent_lines(lines[self.indices], False) == self.expected_lines
+        # OK, one last fallback for Windows...
```

Comment:
Instead of a fallback here, we should force the value of `file` above (where line is None) if the file ends in .rst - line should only ever be None for docstrings.